ebs-snapshot.sh

```diff
@@ -92,8 +93,9 @@ set -o pipefail
 ## Variable Declartions ##
 
 # Get Instance Details
-instance_id=$(wget -q -O- http://169.254.169.254/latest/meta-data/instance-id)
-region=$(wget -q -O- http://169.254.169.254/latest/meta-data/placement/availability-zone | sed -e 's/\([1-9]\).$/\1/g')
+#Uncomment this to determine region automatically
+#region=$(wget -q -O- http://169.254.169.254/latest/meta-data/placement/availability-zone | sed -e 's/\([1-9]\).$/\1/g')
+region=us-west-2
```

Comment:
Is there a reason that it would be preferred to hard-set the region in the script instead of determining it dynamically? Would it be better to set it as a command line parameter and allow the user to pass it in, and if the user doesn't it takes the current region as the default?